packages/app/setup.py

```diff
@@ -48,6 +49,11 @@ def get_var(var_name):
     'galaxy.jobs.runners.util.job_script',
     'galaxy.jobs.splitters',
     'galaxy.managers',
+    # galaxy.model.migrate is in galaxy-data, perhaps the versions should be as well?
+    'galaxy.model.migrate.versions',
```

Comment:
This all belongs in galaxy-data. This just wouldn't work because that isn't a namespace package right.

Reply:
It seems to work - they are different levels of the hierarchy which might be why? I didn't dig into the namespace pacakge PEP to check though. But I like the idea of keeping subcomponents with their parents rather than being so opinionated about what components are strictly part of the "concept" for a given package so I'll move this to galaxy-data.